File: back-end/MS-Auth/auth-service/src/main/java/com/bantads/auth_service/DTOs/ClienteEvent.java
package com.bantads.auth_service.DTOs;

import java.math.BigDecimal;
import java.time.LocalDateTime;

public record ClienteEvent(
    String tipo,
    String cpf,
    String nome,
    String email,
    String telefone,
    BigDecimal salario,
    String status,
    String motivo,
    LocalDateTime dataRejeicao,
    String gerenteCpf
    ) {
}

File: back-end/MS-Conta/conta-service/src/main/kotlin/com/bantads/conta_service/dto/ClienteEvent.kt
package com.bantads.conta_service.dto

import java.math.BigDecimal
import java.time.LocalDateTime

data class ClienteEvent(
    val tipo: String,
    val cpf: String,
    val nome: String?,
    val email: String?,
    val telefone: String?,
    val salario: BigDecimal?,
    val status: String?,
    val motivo: String?,
    val dataRejeicao: LocalDateTime? = null,
    val gerenteCpf: String? = null
)
